pkg/util/cluster.go

```diff
@@ -353,11 +353,17 @@ func (c *ClusterInfoImp) RefreshDeschedulerCR(ctx context.Context, cl client.Cli
 			}
 			return err
 		}
-		if instance.Spec.ProfileCustomizations == nil {
-			misconfiguredDescheduler = true
+
+		// TODO: remove this once deschedulerv1.RelieveAndMigrate will graduate loosing
+		// the dev prefix
+		futureRelieveAndMigrateStable := deschedulerv1.DeschedulerProfile("KubeVirtRelieveAndMigrate")
```

Comment:
Shouldn't it be 
```suggestion
		futureRelieveAndMigrateStable := deschedulerv1.DeschedulerProfile("DevKubeVirtRelieveAndMigrate")
```
?

This is the value in https://github.com/kubevirt/hyperconverged-cluster-operator/blob/703fa5acb0cdb2d280cf1a6a911e64252f49dbbc/vendor/github.com/openshift/cluster-kube-descheduler-operator/pkg/apis/descheduler/v1/types_descheduler.go#L230

Reply:
OK, so it's appears that `deschedulerv1.RelieveAndMigrate` is `"DevKubeVirtRelieveAndMigrate"`.

resolving